### ISSUE TITLE
Continual learning

### ISSUE THREAD

**Issue by Roh-codeur** [dmlc/XGBoost.jl#104]
Hi

Does this package implement "Training continuation"? if so, how can I use it?

Demo for training continuation[](https://xgboost.readthedocs.io/en/latest/python/examples/continuation.html?highlight=xgb_model#demo-for-training-continuation)

thanks
Roh

**Comment by ExpandingMan** [comment]
Yes! as of 2.0 it should be possible to do very much the same thing that is shown in this python example.
